### PR TITLE
feat: validate urls

### PR DESCRIPTION
**Pull request by dplynskiy**
ASP-975

### CODE REVIEW

File: controllers/application.go [hunk 1]
```diff
@@ -298,3 +300,45 @@ func (c *ApiController) DeleteApplication() {
 	c.Data["json"] = wrapActionResponse(object.DeleteApplication(&application))
 	c.ServeJSON()
 }
+
+func (c *ApiController) validateApplicationURLs(application object.Application) {
```
Comment (ISulimanov) [inline controllers/application.go:304]: move to map where key - field, value - error 
than iterate over map and call IsURLValid

File: controllers/provider.go [hunk 2]
```diff
@@ -331,3 +335,55 @@ func (c *ApiController) TestProviderConnection() {
 	}
 	c.ResponseOk()
 }
+
+func (c *ApiController) validateProviderURLs(provider object.Provider) {
```
Comment (ISulimanov) [inline controllers/provider.go:339]: move to map where key - field, value - error 
than iterate over map and call IsURLValid

File: controllers/application.go (hunk 1 above)
Comment (dplynskiy) [reply controllers/application.go:304]: done

File: controllers/provider.go (hunk 2 above)
Comment (dplynskiy) [reply controllers/provider.go:339]: done

**Review by ISulimanov — APPROVED** [review]

**Pull request merged**